custom_model_runner/datarobot_drum/drum/gpu_predictors/nim_predictor.py

```diff
@@ -41,6 +41,17 @@ def __init__(self):
         self.max_model_len = self.get_optional_parameter("NIM_MAX_MODEL_LEN")
         self.log_level = self.get_optional_parameter("NIM_LOG_LEVEL")
 
+        # DR prefix to denote that this is not some standard NIM variable of any sort.
+        self.health_route = self.get_optional_parameter(
+            "DR_NIM_HEALTH_ROUTE", default_value=self.DEFAULT_HEALTH_ROUTE
+        )
+        self.openai_port = self.get_optional_parameter(
+            "DR_NIM_SERVER_PORT", default_value=self.openai_port
+        )
+        self.openai_host = self.get_optional_parameter(
+            "DR_NIM_SERVER_HOST", default_value=self.openai_host
```

Comment:
its already read from env var, but runtime param is a different beast ⭐ as it has a prefix, so for backwards compat we will support both ways.